pattern/script_PhotobleachXYPattern.m

```diff
@@ -20,8 +20,8 @@
     generateXYPattern(true);
 
 % Photobleach configurations
-nPasses = 1; % Keep as low as possible. If galvo gets stuck, increase number
-line_exposure_sec_mm = 0.5; % sec/mm. For gel use 0.5
+nPasses = 2; % For gel use 1 pass. Keep as low as possible. If galvo gets stuck, increase number. 
```

Comment:
State in the comment: "For gel use 1 pass. For brain tissue use 2". Same for the next line

Reply:
Hi I've revised the comments.